strongbox-storage/strongbox-storage-maven-layout-provider/src/test/java/org/carlspring/strongbox/services/ArtifactSearchServiceImplTest.java

```diff
@@ -28,10 +29,10 @@
 
     public static final String REPOSITORYID = "artifact-search-service-test-releases";
 
-    @Inject
+    @Autowired
```

Comment:
@sbespalov : Not merged properly. I dropped all the `@Autowired` in [SB-752](https://youtrack.carlspring.org/issue/SB-752), which was merged into the master as part of [SB-744](https://youtrack.carlspring.org/issue/SB-744).